Dockerfile

```diff
@@ -1,29 +1,42 @@
-# Do not update to Ruby 3 until Gemfile dependencies are fixed
-FROM ruby:2.7.6-alpine3.15
+FROM public.ecr.aws/docker/library/ruby:3.2.3-alpine3.19
 
-# These are needed to support building native extensions during
-# bundle install step
-RUN apk --update add --virtual build_deps build-base git
+LABEL org.opencontainers.image.vendor="Ministry of Justice" \
+      org.opencontainers.image.authors="Operations Engineering" \
+      org.opencontainers.image.title="Tech Docs GitHub Pages Publisher" \
+      org.opencontainers.image.description="Container for build and packaging tech-docs" \
+      org.opencontainers.image.url="https://github.com/ministryofjustice/tech-docs-github-pages-publisher"
 
-RUN addgroup -g 1000 -S appgroup \
-  && adduser -u 1000 --system appuser \
-  && adduser appuser appgroup \
-  && gem install bundler -v 2.4.22\
-  && bundle config
+ENV PUBLISHER_DIRECTORY="/publisher" \
+    BUNDLER_VERSION="2.5.6" \
+    LYCHEE_VERSION="0.14.3"
```

Comment:
I'm unconvinced of the benefit of installing lychee in the container. You can easily make it a step in a GitHub Action and avoid having to maintain the version at source i.e. https://github.com/ministryofjustice/operations-engineering-user-guide/blob/main/.github/workflows/build.yml#L37-L46